recipes/pdb2fasta/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "pdb2fasta" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kad-ecoli/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 63c22998df75c0a1ee9c00051d4d401e45b13651a09b98bb2885a86bc342b7bf
+  patches:
+    - pdb2fasta.patch
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('pdb2fasta', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

Do you need cxx ?
It looks like just C would be enough. 
```suggestion
    - {{ compiler('c') }}
```

---

You're right. In the first version I was trying to compile a `cpp` file, but I ended up changing to a `c` source and forgot to change the compiler.